airflow/serialization/serialized_objects.py

```diff
@@ -433,10 +433,12 @@ def deserialize_operator(cls, encoded_op: Dict[str, Any]) -> BaseOperator:
 
             for ope in plugins_manager.operator_extra_links:
                 for operator in ope.operators:
-                    if (
+                    if ((
                         operator.__name__ == encoded_op["_task_type"]
                         and operator.__module__ == encoded_op["_task_module"]
-                    ):
+                    ) if not isinstance(operator, str) else (
```

Comment:
Please find a cleaner way of writing this -- it is hard/impossible to grok what it is going.

Reply:
Extracted this part to a function in 1fde244c350f302de52ddea92b850ba76f454fcc.